gpfit/fit.py

```diff
@@ -202,5 +202,5 @@ def rfun(p):
         if K == 1:
             cstrt = MonomialEquality(mono2, "=", mono1)
 
-    return cstrt, rmsErr
```

Comment:
Also, when I originally chose to create/return constraints in fit.py, I didn't really think through how it would be used. I doubt people will include raw data directly in GPkit model files. I think the preferred approach is to create the fit offline and then insert it into a GPkit model.